scala/private/rule_impls.bzl

```diff
@@ -127,7 +127,7 @@ touch {statsfile}
         arguments = [],
     )
 
-def _expand_location(ctx, flags):
+def expand_location(ctx, flags):
```

Comment:
Should we move every public method to common.bzl or maybe interim_common.bzl? 
It's somewhat surprising to find a public method in private/rule_impls.bzl

Reply:
I plan to eventually, basically the common functions in rule_impls.bzl will get moved to common.bzl or some other sort of private lib organization. I didn't in these PRs to reduce the size of the incidental diffs during the refactoring process.